### ISSUE TITLE
ng build for lib with self referencing components fails with no message

### ISSUE THREAD

**Issue by MonocleKelso** [angular/angular-cli#11369]
### Bug Report or Feature Request (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
- [ ] feature request
```

### Area
```
- [x] devkit
- [ ] schematics
```

### Versions

Windows 10. Node 9.9.4. NPM 6.1.0

### Repro steps

Create a library and then create two components inside that library. In ComponentA add a reference to ComponentB but change the import statement to reference the library name instead of the relative directory path ala - import {ComponentB} from 'my-lib'. Run a build with ng build and notice that the command line prints Building Angular Package and then exits with no message.

### The log given by the failure

Log simply prints "Building Angular Package" just like command line

### Desired functionality

Some kind of error message. Obviously this is a rather silly error but the first time it happened to me it did take awhile to hunt down what was happening.

**Comment by alan-agius4** [comment]
Which version of ng-packagr are you using? This should have been solved in version 3.0.0.

https://github.com/dherges/ng-packagr/commit/ada4081

**Comment by MonocleKelso** [comment]
Yes. I was on an RC version of 3.0.0.
